.travis.yml

```diff
@@ -7,6 +11,7 @@ services:
 script:
   - bundle exec rspec --format d
   - bundle exec codeclimate-test-reporter
+  - docker build --rm=false . 
```

Comment:
Im curious — why `rm=false`? As far as I know that keeps the container running after building the image.